js/src/views/blog/category_selector.js

```diff
@@ -78,7 +79,10 @@ var CategorySelector = Backbone.View.extend({
   },
 
   onChangeSelect: function(e) {
-    window.location = e.currentTarget.value;
+    var category = e.currentTarget.value;
+
+    window.location.href = category.length > 0 ?
+      'categories#' + category : 'categories';
```

Comment:
I'm not sure if this will work when site base ulr would be like GDA-dashborad/
I think you should add SITEURL here.